contents/blog/what-you-need-to-know-about-open-source-analytics.md

```diff
@@ -0,0 +1,54 @@
+
+## What is open source software?
+Open source is a licensing model which enables anyone to see and modify the source code within the software. This isn’t the case for most software, which is closed source. There are a variety of open-source licenses, but most open source projects are also distributed for free - this is called Free and Open Source Software (FOSS). 
+
+PostHog is an example of an open-source project, as anyone can access our source code via our GitHub repo and we offer a FOSS version of PostHog under a permissive MIT license. This means that the only restriction is that a license and copyright notice must be included with all versions of the software.
+
+## Why is being open-source important in product analytics?
+PostHog’s mission is to help more companies build successful products and the decision to be open source is part of that. We believe we need to change how product analytics works and that the open-source model will enable faster, community-driven progress. 
+
+For example, teams using closed-source analytics tools are often limited on which software they can use or integrate with. An open-source approach removes this issue because teams can add to our source code directly and create new integrations between platforms by themselves. Many popular PostHog plugins were developed by the community, for example. 
+
+An open-source product analytics platform also offers other advantages, such as the ability to influence the product roadmap or enabling users to fix bugs themselves instead of waiting for customer support. If PostHog were to make decisions which the community disagreed with then the users can express that more directly.
+
+PostHog also offers a FOSS version of the software, which makes product analytics accessible to smaller teams or individuals - not just large companies on expensive contracts with closed-source providers.
+
+## What is the best open-source product analytics platform?
+There are many product analytics platforms to choose from but most of these are closed-source platforms, such as MixPanel or Amplitude. However, we believe that PostHog is not just the best open-source product analytics platform, but the best product analytics platform overall. 
+
+This is because, in addition to the advantages which come from an open-source model, PostHog also enables users to deploy onto their existing infrastructure and avoid sharing data with third parties. This results in both easier [compliance with regulations such as HIPAA](/blog/hipaa-compliant-analytics), as well as better data capture. Some users have even found that [PostHog is able to capture up to 50% more events than MixPanel](/customers/saga).
+
+Additionally, PostHog offers a wide range of product analytics tools to help teams get the insights they need. PostHog is the only open-source product analytics platform which offers core analytics tools in addition to features such as session recording and feature flags.
+
```

Comment:
Should we cross link the original blog post?

Reply:
@paolodamico which blog post do you mean?

Reply:
Nvm, I was confused by the original post, I see now it's a Google drive link